apis/v0/plugin_interface.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v0
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/go-plugin"
+	grpc "google.golang.org/grpc"
+)
+
+const (
+	handshakeCookieKey   = "JVS_PLUGIN"
+	handshakeCookieValue = "hello"
```

Comment:
Usually the cookie value is a long string. It's not meant for security, but "hello" is too generic. `openssl rand -hex 30` will give you a good value.

Reply:
Updated :D